inference.py

```diff
@@ -43,112 +45,223 @@
 logging.getLogger(__name__)
 
 
-def calc_inference_chunk_size(gpu_devices_dict: dict, max_pix_per_mb_gpu: int = 350):
```

Comment:
why remove this function? Did it cause problems?

Reply:
Yeah, it does not work well with the revised smoothening function. I can explain further. It has to do with the padding of the image x2 of its dimension, so you have to provide a static chunk_size which will not burst the memory during operations.